cli/actions/generation.go

```diff
@@ -86,5 +86,34 @@ func (s *sedgeActions) Generate(options GenerateOptions) (generate.GenData, erro
 	}
 	log.Info(configs.CleanedGeneratedFiles)
 
+	// create datadir folders
+	datadirs := []struct {
+		path     string
+		createIf bool
+	}{
+		{
+			path:     filepath.Join(options.GenerationPath, configs.ExecutionDir),
+			createIf: options.GenerationData.ExecutionClient != nil,
+		},
+		{
+			path:     filepath.Join(options.GenerationPath, configs.ConsensusDir),
+			createIf: options.GenerationData.ConsensusClient != nil,
+		},
+		{
+			path:     filepath.Join(options.GenerationPath, configs.ValidatorDir),
+			createIf: options.GenerationData.ValidatorClient != nil,
+		},
+	}
+	for _, datadir := range datadirs {
+		if datadir.createIf {
+			if _, err := os.Stat(datadir.path); os.IsNotExist(err) {
```

Comment:
You are ignoring `err` here except when it is an `os.IsNotExist`. Is this intentional? There could be another type of error related to something else.

Reply:
Fixed in 1741c7fb72189f23933f99ef0e8c1657ee643598